yt/data_objects/tests/test_numpy_ops.py

```diff
@@ -123,11 +123,19 @@ def test_min_max():
         )
 
         p = ad.max(("gas", "density"), axis=1)
-        p1 = ds.proj(("gas", "density"), 1, data_source=ad, method="mip")
+        p1 = ds.proj(("gas", "density"), 1, data_source=ad, method="max")
+        assert_equal(p[("gas", "density")], p1[("gas", "density")])
+
+        p = ad.min(("gas", "density"), axis=1)
+        p1 = ds.proj(("gas", "density"), 1, data_source=ad, method="min")
```

Comment:
Here you're hiding a `NotImplementedError` from https://github.com/yt-project/yt/blob/0f9d2f4eb9625d6aea54f1d16fde7bb54ad9fb70/yt/data_objects/data_containers.py#L1043
which I'm guessing needs to be replaced with actual code. Taking a look at the similar `max` method it should be pretty trivial.

Reply:
Done!